src/components/enterprise-user-subsidy/enterprise-offers/data/hooks.js

```diff
@@ -47,23 +63,30 @@ export function useEnterpriseOffers({
       }
     };
 
-    if (features.FEATURE_ENROLL_WITH_ENTERPRISE_OFFERS) {
+    if (enableOffers) {
       fetchEnterpriseCoupons();
     } else {
       setIsLoadingEnterpriseCoupons(false);
     }
   }, [enterpriseId]);
 
   useEffect(() => {
-    if (isLoading || !features.FEATURE_ENROLL_WITH_ENTERPRISE_OFFERS) {
+    if (!enableOffers || isLoading) {
       return;
     }
 
-    // might also add enterpriseOffers.length === 1 depending on requirements
-    if (enterpriseCoupons.length === 0
-       && (customerAgreementConfig?.subscriptions?.length || 0) === 0) {
-      setCanEnrollWithEnterpriseOffers(true);
+    const enterpriseHasCoupons = enterpriseCoupons.length > 0;
+    const enterpriseHasSubscriptions = (customerAgreementConfig?.subscriptions?.length || 0) > 0;
+
+    // For MVP we will only support enterprises with one active offer
+    const enterpriseHasOneOffer = enterpriseOffers.length === 1;
+
+    if (enterpriseHasCoupons || enterpriseHasSubscriptions || !enterpriseHasOneOffer) {
+      return;
     }
+
+    setCanEnrollWithEnterpriseOffers(true);
+    setHasLowEnterpriseOffersBalance(enterpriseOffers[0].remainingBalance <= ENTERPRISE_OFFER_LOW_BALANCE_THRESHOLD);
```

Comment:
Do we need to add `enableLearnerPortalOffers` to the dependency array here?

Reply:
yeah good catch, once we upgrade to front-end build with the correct eslint rules, these will be caught automatically